master/buildbot/status/builder.py

```diff
@@ -322,12 +328,27 @@ def getLastFinishedBuild(self):
             b = self.getBuild(-2)
         return b
 
-    def setCategory(self, category):
+    def getTags(self):
+        return self.tags
+
+    def setTags(self, tags):
         # used during reconfig
-        self.category = category
+        self.tags = tags
+
+    def matchesAnyTag(self, tags):
+        return self.tags and any((tag in self.tags) for tag in tags)
```

Comment:
If either parameter `tags` or `self.tags` is empty, the result is `False`, this is expected, right?